src/lib.rs

```diff
@@ -89,6 +89,9 @@ mod error;
 mod integer;
 mod uid;
 mod value;
+#[macro_use]
+#[cfg(feature = "serde")]
```

Comment:
This doesn't look like it uses any serde features, so I don't think needs the `cfg`.